### PR TITLE
UC6.5 next revision

### PR DESCRIPTION
**Pull request by eric-schuh**
Cleanup and updates to the first half from a review w/Joe. Questions coming out of that review are:

1. Ok to assume CHAPI for the user-browser-wallet interactions?
    1. (We are assuming yes and moving forward with the diagrams using CHAPI unless we hear otherwise)
1. How does the user authenticate, i.e. your “Initiate Session” in the [linked diagram](https://docs.google.com/document/d/1-u0_Ub6feiX6DH3jXFJFjt6n3CwKGpkmC3VISqDkWL4/edit#heading=h.naddhxq5s5pq), to the two websites in this use case?
    1. Do they just login with existing classical login credentials?
    1. Are they using a provided code to collect the credential (in the vein of the Digital Bazaar USCIC Permanent Residents Cards use case)?
    1. Do they do a DID Auth, to a DID already known by the particular website?
1. What is your opinion on whether this diagram is showing the flows synchronously or implying asynchronous operations? 
    1. Some actions must be async but there are a number of the webpage interactions that could be modeled either way and we would like the diagram to represent how you were best thinking about this flow.

Image: 

image

***
<a href="https://pr-preview.s3.amazonaws.com/eric-schuh/vc-api-use-cases/pull/20.html" title="Last updated on Aug 30, 2022, 7:39 PM UTC (7a7845f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api-use-cases/20/a163df1...eric-schuh:7a7845f.html" title="Last updated on Aug 30, 2022, 7:39 PM UTC (7a7845f)">Diff</a>

### CODE REVIEW

**Comment by eric-schuh** [comment]
@martyr160

Can you review the changes and questions (more to come after this Friday when Joe and I get around to reviewing the second half). Any comments/questions for us are welcome!

**Comment by eric-schuh** [comment]
@martyr160 ping in case you missed this, your feedback would be appreciated.

**Comment by TallTed** [comment]
@eric-schuh -- You seem to have the wrong GitHub handle for whomever you meant by [`@martyr160`](https://github.com/martyr160).  Possibly you meant [`@martyr280`](https://github.com/martyr280) (who I've not tagged, only linked, in case I've guessed wrong)?

**Comment by eric-schuh** [comment]
@TallTed Yes, you seem to be correct. Apparently got the wrong handle from Joe. Thanks for the Clarification

@martyr280 If you have a moment mind taking a look at the top comment in the PR? Joe and I had some questions for you as the one who submitted this Use Case.

File: index.html [hunk 1]
```diff
@@ -473,63 +473,63 @@ <h3>Submit/Sign/Verify a test credential to a licensure system</h3>
...
+                    Shabazz's Wallet <br/>(Holder Service)->>Shabazz's Billfold <br/>(Holder App - includes browser):VP w/Application and State License
+                    Shabazz's Billfold <br/>(Holder App - includes browser)->>floydsmassagetheray.com <br/>(Verifier App):floydsmessagetherapy.com/presenatations/submit?"Application wtih State License VP"
+                    floydsmassagetheray.com <br/>(Verifier App)->>State Verifier (Verifier Service):stateverifier.com/credentials/verify?"StateLicenseVC"
+                    State Verifier (Verifier Service)->>floydsmassagetheray.com <br/>(Verifier App):Verification Result
+                    floydsmassagetheray.com <br/>(Verifier App)->>floydsmassagetheray.com <br/>(Verifier App):process application
+                    floydsmassagetheray.com <br/>(Verifier App)->>Shabazz's Billfold <br/>(Holder App - includes browser):Application submitted
+                    Shabazz's Billfold <br/>(Holder App - includes browser)->>Shabazz <br/>(Holder):Application submitted
           </pre>
           <figcaption>UC2.5 Mermaid</figcaption>
```
Comment (TallTed) [inline index.html:534]:
```suggestion
          <figcaption>UC6.5 Mermaid</figcaption>
```

**Comment by TallTed** [comment]
@jandrieu -- Merged without correction to `figcaption`!

**Comment by eric-schuh** [comment]
@TallTed this fix got in, just not in this PR. Live index.html has the correct "UC6.5"

**Pull request merged**